README.md

```diff
@@ -48,7 +48,7 @@ EOF
 
 ### In the browser
 Use the following [bookmarklet](https://en.wikipedia.org/wiki/Bookmarklet) that's powered by [BootstrapCDN](http://www.bootstrapcdn.com/#bootlint_tab):
-```
+```javascript
```

Comment:
@hnrch02 based on https://raw.githubusercontent.com/twbs/bootlint/master/README.md you ```javascript for the heading already in the Node.js section. Would you like both switched or just leave it as it was?